srcpkgs/firejail/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'firejail'
 pkgname=firejail
-version=0.9.56
-revision=2
+version=0.9.58
+revision=1
 build_style=gnu-configure
 configure_args="--enable-apparmor"
+conf_files="/etc/firejail/* /etc/apparmor.d/local/firejail-local"
```

Comment:
Its really not necessary to move this around. https://github.com/chneukirchen/xtools/pull/115

Reply:
Ah, just trying to silence xlint.